KenticoOnboardingApplication/KenticoOnboardingApplication.Api.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace KenticoOnboardingApplication.Api.Tests
+{
+    [TestClass]
+    public class UnitTest1
```

Comment:
It is only a dummy test ... but even the dummy test deserves a better name. Think what is this test good for ...